src/layouts/ConnectionLayout.astro

```diff
@@ -0,0 +1,22 @@
+---
+import BaseLayout from './BaseLayout.astro';
+import Farmlands from '../components/Farmlands.vue';
+import GuideLayout from '../components/GuideLayout.vue';
+
+const img = 'countryside.jpg';
+const fullWidth = true;
+const headings = {};
+---
+
+<BaseLayout>
+  <Farmlands/>
+    <GuideLayout
```

Comment:
I think adding variants to the guide is going to couple the display of a guide and cover pages together.
Was there a suggestion from Tom to break this out into its own layout?

Reply:
As per my comment to Tom I have added this small change and will create another layout in a follow up PR